aten/src/ATen/Version.cpp

```diff
@@ -97,6 +97,14 @@ std::string used_cpu_capability() {
   ss << "CPU capability usage: ";
   auto capability = native::get_cpu_capability();
   switch (capability) {
+#ifdef HAVE_VSX_CPU_DEFINITION
```

Comment:
@VitalyFedyunin sorry, you mean I should revert it back, right? as I understand you want to work on these details on your own

Reply:
https://github.com/pytorch/pytorch/pull/41541/files#r491377061

Reply:
@VitalyFedyunin sorry man. I think if I remove it there I have to remove architecture additions, too. If you have free time, we could discuss it. Maybe I can do how you desire those things to be. 
thanks

Reply:
@quickwritereader I'm moving today and tomorrow, so there is limited access to the internet and no laptop. I would be able to chat with you (btw do you have access to the pytorch slack?) this Thursday afternoon (or your preferred time as i'm in EST zone). Meanwhile I encourage you to rebase this PR on top of tests and if possible get rid of sleef update. Also I think that we can take this capability changes as is. As they are require proper implementation which is out of scope for this PR. But don't hold me for this.

Reply:
@VitalyFedyunin no I don't have an access to the PyTorch slack.
yes, it suits me if you take the current changes as is. ( Because of no coordination I had to make a minimal invasion and not changing arch handling).